server/src/reactruntime.js

```diff
@@ -3,6 +3,8 @@ require("core-js");
 const React = require("react");
 const ReactDOM = require("react-dom");
 const linker = require("./linker");
+const { LocalizationProvider } = require("fluent-react/compat");
+const l10n = require("./l10n");
```

Comment:
Ah, this is the bad import!  reactruntime is used in the browser, and so it should only import modules that are okay to have in the browser, and `./l10n` is not.  Instead we should be injecting the output of l10n into the module somehow.